hopsandcp/dcpmaster.cpp

```diff
@@ -0,0 +1,259 @@
+#include "dcpmaster.h"
+
+#include <iostream>
+#include <cstdint>
+#include <fstream>
+#include <memory>
+
+#include "dcp/helper/LogHelper.hpp"
+#include "dcp/model/pdu/DcpPduFactory.hpp"
+#include "dcp/xml/DcpSlaveDescriptionReader.hpp"
+#include "dcp/driver/ethernet/udp/UdpDriver.hpp"
+#include "dcp/logic/DcpManagerMaster.hpp"
+
+
+DcpMaster::DcpMaster(const std::string host, u_short port)
```

Comment:
Calling stuff Master/Slave is discouraged nowdays. But since they are called master/slave in the dcp library I understand why you did. Not sure if we should call them something else or just leave the names.
Maybe Master/Worker, not sure

Reply:
You are right, I actually did not hear of this before. It is not only the DCP library that uses it. This terminology is very strong in the co-simulation field. The DCP standard uses the word "slave" 497 times in their specification, and the FMI standard 130 times. But I guess we should call it something else. I don't really like "worker", since it is actually a connection to an something external. Could we call it "DCPServer"? Or maybe "DCPConnection"?

Reply:
Apparantly "Master" is also discouraged, but I guess it depends more on the context. We could call it "DCPManager" instead, which is actually more accurate description of what it does.

Reply:
Unfortunately I think it would be best to keep the master/slave names to avoid confusion. If out classes match DCP master/slave classes. Otherwise rename them. But yes maybe Manager/Worker, I dont know.